### PR TITLE
[load-theme-styles] Add es6 "module" entry in package.json

### LINKED ISSUE #1292: [load-theme-styles] Load theme styles package contains only commonjs output
We'd love if this package included ESM in addition to (or instead of) the commonjs output so that it is eligible for webpack optimizations like tree shaking or module concatenation

### PR DESCRIPTION
**Pull request by kenotron**
Fixes #1292

Simply adds a package.json entry for load-theme-styles to make esm friendly bundlers happy

### CODE REVIEW

File: common/changes/@microsoft/load-themed-styles/es6_2019-12-17-05-40.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/load-themed-styles",
+      "comment": "adds es6 library for load-theme-styles",
```
Comment (iclanton) [inline common/changes/@microsoft/load-themed-styles/es6_2019-12-17-05-40.json]:
```suggestion
      "comment": "Add a \"module\" entry in package.json for the es6-style library.",
```

Comment (octogonz) [reply common/changes/@microsoft/load-themed-styles/es6_2019-12-17-05-40.json]: @kenotron I tried to accept this change, but it seems your branch is protected against commits from maintainers.

Comment (kenotron) [reply common/changes/@microsoft/load-themed-styles/es6_2019-12-17-05-40.json]: image

Comment (kenotron) [reply common/changes/@microsoft/load-themed-styles/es6_2019-12-17-05-40.json]: I'll just accept ian's suggestion real quick!

**Comment by kenotron** [comment]
thx!

**Pull request merged**